pkg/operator/ceph/object/shared_pools_test.go

```diff
@@ -433,37 +467,12 @@ func Test_toZonePlacementPool(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "map default placement without extra SC to config",
-			args: args{
-				spec: cephv1.PoolPlacementSpec{
-					Name:              defaultPlacementName,
-					MetadataPoolName:  "meta",
-					DataPoolName:      "data",
-					DataNonECPoolName: "repl",
-					StorageClasses:    []cephv1.PlacementStorageClassSpec{},
-				},
-				ns: "ns",
-			},
-			want: ZonePlacementPool{
-				Key: defaultPlacementCephConfigName,
-				Val: ZonePlacementPoolVal{
-					DataExtraPool: "repl:ns.data.non-ec",
-					IndexPool:     "meta:ns.index",
-					StorageClasses: map[string]ZonePlacementStorageClass{
-						defaultPlacementStorageClass: {
-							DataPool: "data:ns.data",
-						},
-					},
-					InlineData: true,
-				},
-			},
-		},
```

Comment:
Is there a reason this test is being removed? Is it no longer applicable?

Reply:
i rewieved all test-cases during development of this feature and discovered that this test-case was duplicate.